applications/FluidDynamicsApplication/python_scripts/navier_stokes_ale_fluid_solver.py

```diff
@@ -4,9 +4,11 @@
 import KratosMultiphysics
 
 # other imports
-from KratosMultiphysics.MeshMovingApplication.ale_fluid_solver import AleFluidSolver
 import KratosMultiphysics.FluidDynamicsApplication.python_solvers_wrapper_fluid as fluid_solvers_wrapper
 
+have_mesh_moving = KratosUtilities.CheckIfApplicationsAvailable("MeshMovingApplication")
+if have_mesh_moving:
+    from KratosMultiphysics.MeshMovingApplication.ale_fluid_solver import AleFluidSolver
```

Comment:
@philbucher should we throw an error here, if we want to use the Navier-Stokes ALE fluid  the MeshMovingApplication is not available?

Reply:
yes, wont work otherwise